### ISSUE TITLE
TypeError: Uncaught error: reply.view is not a function

### ISSUE THREAD

**Issue by nelsonic** [dwyl/hapi-error#49]
It's pretty bad when your "friendly" `error` reporting module is _throwing_ an "implementation error" ... 😕 

![image](https://user-images.githubusercontent.com/194400/28096170-9abee370-669e-11e7-918e-3f212a6e533f.png)

Full stack trace:
```
Debug: internal, implementation, error 
    TypeError: Uncaught error: reply.view is not a function
    at /dwyl/hapi-socketio-redis-chat-example/node_modules/hapi-error/lib/index.js:153:20
    at each (/dwyl/hapi-socketio-redis-chat-example/node_modules/hapi/lib/request.js:430:22)
    at iterate (/dwyl/hapi-socketio-redis-chat-example/node_modules/items/lib/index.js:36:13)
    at Object.exports.serial (/dwyl/hapi-socketio-redis-chat-example/node_modules/items/lib/index.js:39:9)
    at _protect.run (/dwyl/hapi-socketio-redis-chat-example/node_modules/hapi/lib/request.js:433:15)
    at internals.Protect.run (/dwyl/hapi-socketio-redis-chat-example/node_modules/hapi/lib/protect.js:59:12)
    at internals.Request._invoke (/dwyl/hapi-socketio-redis-chat-example/node_modules/hapi/lib/request.js:413:19)
    at internals.Request._reply (/dwyl/hapi-socketio-redis-chat-example/node_modules/hapi/lib/request.js:481:17)
    at Items.serial (/dwyl/hapi-socketio-redis-chat-example/node_modules/hapi/lib/request.js:407:65)
    at done (/dwyl/hapi-socketio-redis-chat-example/node_modules/items/lib/index.js:31:25)
```
Context: https://github.com/dwyl/hapi-socketio-redis-chat-example/pull/91#issuecomment-314250774

we can fix this by adding a check (`if` statement) however the _test_ is not quite as easy ...

**Comment by nelsonic** [comment]
Totes done in less than `T1h` ✅

**Comment by nelsonic** [comment]
Fixed in #51
